### PR TITLE
Update shuttles and shuttle floors for in rotation maps

### PR DESCRIPTION
**Pull request by CheffieGithub**
## About the PR 
Replace icon varedits with the typed colours for shuttle floors.
Removes turf shuttle corners from all arrival shuttles.
Changes dir of shuttle floors to match arrival shuttle direction where it doesn't.
Puts plating under all windows in arrival shuttles.
Replace transparent broken walls with windows in shuttle cockpits.
Update wrestlemap arrival shutttle with compact sleepers.
Removed shuttle corners from the horizon ranch shuttle on donut 2.

Update the old heater in the cog 1 wreck to the newer ones:
image

Slight moving around of the donut 3 arrival shuttle:
image
(lights and cyro moved)

Removed the extra windows and extended the donut 2 arrival shuttle and area:
image

## Why's this needed? 
Fixes and updates for shuttles. Good for consistency. Typed floors will be better if anyone changes the icon_state.

### CODE REVIEW

**Comment by ZeWaka** [comment]
> donut 2 arrival shuttle:

i feel like cyro should be moved up one tile...

**Comment by CheffieGithub** [comment]
Not really possible due to heater and wingrilles. Getting the layering to look right will probably cause issues.
image

Instead I extended arrivals by one tile as a whole so the shuttle can have some space for cyro without seats in the way.

**Comment by flappybatpal** [comment]
This is getting out of scope but there are a number of things about shuttles I'd question

1. I never really understood the design decision to put the heaters over wingrilles.

image

I have put the left side over shuttle walls, the right side are over windows. I think it might be a carry over from the old heater sprites that do not cover nearly the entire tile.
The heaters can be exploded but the propulsion units are invincible so you can't move past them even if the window and heater is gone.
2. Not sure why the shuttle walls are simulated rather than unsimulated, although it is very rare that they take damage outside of meteors. The windows can interactive for the rare case where it might be unsafe to leave such as blob or flood.
3. We should delete and replace the old propulsion and heater sprites. The fact these are varedited on every map is silly and we shouldn't be using perspective incorrect icons elsewhere.

**Comment by CheffieGithub** [comment]
I think the point of wingrilles is to allow you to see through the back, lots of walls on shuttles are varedited to be transparent like the cockpit walls and the old shuttle windows (which are just transparent old flat walls).
I don't know why they are simulated either since they have return cases for almost all types of damage its been that way for a while. I can replace them in this PR or a different one if you like.

**Comment by Sord213** [comment]
I'm guessing the shuttle walls are just super duper old, probably before we had unsimulated walls maybe? It confused me as well but effectively doesn't make much of a difference. Either way changing all the janky old shuttle stuff is probably out of scope for this PR, but it's something we could (and probably should) change....at some point....later....

**Comment by flappybatpal** [comment]
> I think the point of wingrilles is to allow you to see through the back, lots of walls on shuttles are varedited to be transparent like the cockpit walls and the old shuttle windows (which are just transparent old flat walls). I don't know why they are simulated either since they have return cases for almost all types of damage its been that way for a while. I can replace them in this PR or a different one if you like.

image

That does rely on the propulsion not being opaque and they have a default opacity of 1.

**Comment by flappybatpal** [comment]
image

You can get under these on Cogmap2, Donut2 and Donut 3 (including asylum shuttle) and Wrestlemap.

**Review by flappybatpal — CHANGES_REQUESTED** [review]

File: maps/kondaru.dmm [hunk 1]
```diff
@@ -23,10 +23,11 @@
 /turf/space,
 /area/space)
 "aag" = (
-/turf/simulated/shuttle/wall{
-	dir = 8;
-	icon_state = "wall3_space"
+/obj/indestructible/shuttle_corner{
+	icon_state = "wall3_trans";
```
[the remark this answers is not in the document]
Comment (CheffieGithub) [reply maps/kondaru.dmm:27]: Because all the shuttle corners are varedits, I plan to change that but not in this PR

**Comment by flappybatpal** [comment]
Considering another PR is planned I will merge as-is.

**Pull request merged**